.vsts-dotnet-ci.yml

```diff
@@ -0,0 +1,146 @@
+trigger:
+- master
+- exp/*
+
+phases:
+- phase: FullOnWindows
+  displayName: "Build on Windows using full MSBuild"
+  queue: 'Hosted VS2017'
+  steps:
+  - script: set
+    displayName: Command Line Script
+  - task: BatchScript@1
+    displayName: VsDevCmd
+    inputs:
+      filename: 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat'
+      modifyEnvironment: true
+  - task: BatchScript@1
+    displayName: cibuild.cmd
+    timeoutInMinutes: 60
```

Comment:
I think the default timeout for steps is infinite.

The default for the phase is 60, not sure if you need to explicitly set this here - https://github.com/Microsoft/vsts-agent/blob/master/docs/preview/yamlgettingstarted-job.md#timeout

Reply:
I think I couldn't figure out how to set it explicitly on phase so I set it on the part that could hang. Removed now; we can adjust later.